### ISSUE TITLE
Did I miss it somewhere?  Where do you say the prefered way to step through code?  Not step over, or into, or whatever.  Just preferred method to step through line by line.  I think I must have missed it?

### ISSUE THREAD

**Issue by ArloWeston** [google/WebFundamentals#6724]
**Welcome!** Please use this template for reporting documentation issues on https://developers.google.com/web/.

_For questions about **Chrome DevTools**, checkout the
[`google-chrome-devtools`](https://stackoverflow.com/questions/tagged/google-chrome-devtools) tag on StackOverflow.
For questions about **Chrome**, check the [`google-chrome`](https://stackoverflow.com/questions/tagged/google-chrome)
tag. If you think you've found a Chrome bug, check out our 
[feedback page](https://developers.google.com/web/feedback/#how-to-file-a-good-bug) for more details on how to file it._

**Page Affected:** https://developers.google.com/web/TODO

**What needs to be done?**

_Please provide description of what's needs to be done here. Please be sure to include the link to the original page
and any supporting links/data._

- [ ] Appropriate `type-something` label has been added.
- [ ] Appropriate `section-something` label has been added.

**Comment by kaycebasques** [comment]
Which doc are you referring to?

**Comment by ArloWeston** [comment]
Sorry, I'm going to close this issue.  It was my mistake.  When I was debugging, the focus was on the web page, not the developer tools.  When I clicked F11 it maximized the page, not stepped through the code.  I thought Google had updated it so F11 didn't work anymore to step line by line through the JavaScript.  I figured it out the next day.  I forgot this was still open.

One thing, someone might want to put in the documentation that F11 will step through the code line by line, after it is stopped on a break point.
